slack-notify/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Slack Notification
+
+inputs:
+  slack-notify-override:
+    description: "Send the notification regardless of success or failure"
+    default: false
+    required: false
+    type: boolean
+  slack-webhook-url:
```

Comment:
Because this is a "slack-notify" action, the inputs being prefixed with "slack_" seems odd to me 🤔

Reply:
That makes sense since we already know the context, I'll get this updated